libraries/azure-app-configuration-importer/src/featureFlag.ts

```diff
@@ -0,0 +1,61 @@
+import { ClientFilter } from "./models";
+import { FeatureFlagValue as FeatureFlagValue1 } from "@azure/app-configuration";
+
+export interface FeatureFlagValue extends FeatureFlagValue1 {
+    id?: string;
+    description?: string;
```

Comment:
We don't have to repeat the field which already be declared in `FeatureFlagValue1`

Reply:
You are right, updated